deploy/packages/deb/debian/postrm

```diff
@@ -58,6 +55,11 @@ case "$1" in
         if [ -d /usr/lib/emqx ]; then
                 rm -r /usr/lib/emqx
         fi
+
+        if pidof systemd; then
```

Comment:
Should this be `systemctl status --no-pager`?

I think that in `postrm` for 5.0 it is also using `pidof systemd`.